lms/djangoapps/django_comment_client/permissions.py

```diff
@@ -55,10 +56,35 @@ def check_question_author(user, content):
         except KeyError:
             return False
 
+    def check_team_member(user, content):
+        """
+        If the content has a commentable_id, verifies that either it is not associated with a team,
+        or if it is, that the user is a member of that team.
+        """
+        if not content:
+            return False
+        try:
+            commentable_id = content['commentable_id']
```

Comment:
Actually, it turns out that I can't rely on context because Commentable isn't something we actually get from the comment service. And during the rendering process, we are going to need to know if a user has permissions to post a new thread to a Commentable, so I can't avoid the Teams query at some point, and given the request caching, it doesn't make sense to try to shortcut other calls.